libs/content/workspace-ffi/src/apple/ios_ffi.rs

```diff
@@ -810,7 +810,7 @@ pub unsafe extern "C" fn current_tab(obj: *mut c_void) -> i64 {
             Some(tab) => match tab {
                 TabContent::Image(_) => 2,
                 TabContent::Markdown(_) => 3,
-                TabContent::PlainText(_) => 4,
+                // TabContent::PlainText(_) => 4,
```

Comment:
what's happening here, what even is this function @smailbarkouch

Reply:
This lets the apple clients know what tab type is open.